### ISSUE TITLE
libnatpmp fails to install required header

### ISSUE THREAD

**Issue by stefantalpalaru** [Homebrew/homebrew-core#129653]
### `brew gist-logs <formula>` link OR `brew config` AND `brew doctor` output

```shell
user@users-iMac-Pro ~ % brew config
HOMEBREW_VERSION: 4.0.15
ORIGIN: https://github.com/Homebrew/brew
HEAD: 763c41f006ab9b44261e313b0aaa24a047431b06
Last commit: 4 days ago
Core tap JSON: 28 Apr 17:32 UTC
HOMEBREW_PREFIX: /usr/local
HOMEBREW_CASK_OPTS: []
HOMEBREW_MAKE_JOBS: 16
HOMEBREW_NO_ANALYTICS: set
Homebrew Ruby: 2.6.10 => /usr/local/Homebrew/Library/Homebrew/vendor/portable-ruby/2.6.10_1/bin/ruby
CPU: 16-core 64-bit penryn
Clang: 13.0.0 build 1300
Git: 2.32.0 => /Applications/Xcode.app/Contents/Developer/usr/bin/git
Curl: 7.64.1 => /usr/bin/curl
macOS: 11.7.4-x86_64
CLT: 13.2.0.0.1.1637719136
Xcode: 13.2.1

user@users-iMac-Pro ~ % brew doctor
Your system is ready to brew.
```

### Verification

- [X] My "`brew doctor` output" says `Your system is ready to brew.` and am still able to reproduce my issue.
- [X] I ran `brew update` and am still able to reproduce my issue.
- [X] I have resolved all warnings from `brew doctor` and that did not fix my problem.
- [X] I searched for recent similar issues at https://github.com/Homebrew/homebrew-core/issues?q=is%3Aissue and found no duplicates.

### What were you trying to do (and why)?

I am building software that uses libnatpmp, but this package is missing a header.

### What happened (include all command output)?

```text
/usr/local/include/natpmp.h:52:10: fatal error: 'natpmp_declspec.h' file not found
```

This problem was fixed upstream on Jan 12, 2022: https://github.com/miniupnp/libnatpmp/commit/5f4a7c65837a56e62c133db33c28cd1ea71db662

but, for some strange reason, the fix is not in http://miniupnp.free.fr/files/download.php?file=libnatpmp-20230423.tar.gz which is used by the Homebrew formula. Did upstream pack the wrong Git commit?

### What did you expect to happen?

"natpmp_declspec.h" should be installed by the "libnatpmp" package. It's missing:

```text
user@users-iMac-Pro ~ % brew ls -v libnatpmp
/usr/local/Cellar/libnatpmp/20230423/INSTALL_RECEIPT.json
/usr/local/Cellar/libnatpmp/20230423/LICENSE
/usr/local/Cellar/libnatpmp/20230423/bin/natpmpc
/usr/local/Cellar/libnatpmp/20230423/.brew/libnatpmp.rb
/usr/local/Cellar/libnatpmp/20230423/include/natpmp.h
/usr/local/Cellar/libnatpmp/20230423/README
/usr/local/Cellar/libnatpmp/20230423/lib/libnatpmp.1.dylib
/usr/local/Cellar/libnatpmp/20230423/lib/libnatpmp.a
/usr/local/Cellar/libnatpmp/20230423/lib/libnatpmp.dylib
/usr/local/Cellar/libnatpmp/20230423/Changelog.txt
```

### Step-by-step reproduction instructions (by running `brew` commands)

```shell
% brew install libnatpmp
% ls -l /usr/local/include/natpmp_declspec.h
```

**Comment by ZhongRuoyu** [comment]
The fix #129670 is now merged. Please try `brew reinstall libnatpmp`. (If you do this right now, please wait for up to about half an hour.)

**Comment by stefantalpalaru** [comment]
Works now. Thanks!
